content/blog/2020/07/2020-07-24-custom-distribution-service-phase-2.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Custom Distribution Service : Phase 2 Blogpost"
+tags:
+- service
+- distribution
+- cloud-native
+- gsoc
+- gsoc2020
+- packaging
+- platform-sig
+author: sladyn98
+opengraph:
+  image: /images/gsoc/cds_gsoc.png
+---
+
+
+Hello everyone,
+It is time to wrap up another successfull phase for the link:https://github.com/jenkinsci/custom-distribution-service[cutsom distribution service project], we have incorporated most of the features that we had planned at the start of the phase. It has been an immense learning curve for me and the entire team.
+
+To understand what the project is about and the past progress, please refer to the phase one blogpost
+
+== New Features Added
+
+We targeted three major features for this phase namely the addition of community configurations, war download feature, and the ability to filter plugins. Additional minor features included the searching of community configurations as well as the ability to create pull requests from the front-end of the service.
```

Comment:
I think you can also talk about how to "get started" with the application too.

Pictures of the new features are nice as well.

Reply:
Sounds cool. Will add it today

Reply:
I think the get started guide can be included in the readme with the developers guide rather than in the blogpost